data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
             updatedBy: "Alexander", updatedAt: "2020-02-03",
         },
     },
+    {
+        meta: {
+            name: "12/06/2020 Olympia",
+            link: "https://www.startnext.com/12062020",
+            keywords: ["petition", "networking", "event", "grassroots"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "One-time event where up to 90,000+ people come together to rent the Olympia stadium Berlin to sign various petitions."
+            + " In Germany, petitions must be discussed by parliament if they get more than 50,000 signatures. In addition there will be talks and presentations on currently pressing social and environmental problems.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "German Zero",
+            link: "https://germanzero.de/",
+            keywords: ["policy", "legislation"],
+            label: "12-06-2020-olympia",
+        },
+        description: {
+            content: "GermanZero is creating a climate action plan to meet Germany's 1.5C commitment that will be turned into proposed legislation before 2022. A nation-wide campaign will then try to get that legislation passed in parliament.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
```

Comment:
wrong link

Reply:
What's wrong about it? What would be a right link?

Reply:
because it is the link to the previous organization lol
This is the link https://germanzero.de/werwirsind we are searching for.
Still everthing is in german. Might this be a problem?